action.yaml

```diff
@@ -412,6 +404,12 @@ runs:
         else
           sonar_result="SonarQube"
         fi
+
+        if [ "${{ github.event_name }}" == "pull_request" ] || [ "${{ github.event_name }}" == "pull_request_target" ]; then
```

Comment:
In [here](https://github.com/lumada-common-services/gh-composite-actions/pull/112/files#diff-fab4d7fb461bc6fbe9587f6c03fff98102b1c744145edcf2a993f2ff7cb05a0dL394), we were only displaying the URL when the execution was successful, your change make the URL always present even when there might not be an URL to display.

Reply:
updated